lib/src/main/java/tech/relaycorp/awaladroid/endpoint/FirstPartyEndpoint.kt

```diff
@@ -282,10 +285,14 @@ internal constructor(
             } catch (exc: KeyStoreBackendException) {
                 throw PersistenceException("Failed to load certificate for endpoint", exc)
             }
+
+            val internetAddress: String? = context.storage.internetAddress.get()
```

Comment:
I think we should be raising a `PersistenceException` when this value is missing, as done in the lines immediate above.

Reply:
Also, please make sure to add a new test for that.

Reply:
But can't the value be missing when its null?

Reply:
Reopening this because it wasn't fixed in the last commit (maybe you forgot to push it @Filmaluco?)

Reply:
I meant the test btw